<launch>
	<node pkg="prometheus_demo" type="siamrpn_track" name="siamrpn_track" output="screen">
		<param name="uav_id" value="1" />
		<!-- 视觉丢失次数阈值 -->
		<param name="Thres_vision" value="10" />

		<!-- 跟踪到前后间隔 -->
		<param name="tracking_delta_x" value="2" />
		<param name="tracking_delta_y" value="0" />
		<param name="tracking_delta_z" value="1" />

		<!-- 相机相对于无人机偏移 -->
		<param name="camera_offset_x" value="0.01" />
		<param name="camera_offset_y" value="0" />
		<param name="camera_offset_z" value="0" />

		<!-- 跟踪系数 -->
		<param name="kpx_track" value="0.2" />
		<param name="kpy_track" value="0.2" />
		<param name="kpz_track" value="0.2" />

	</node>
</launch>